roles/operator_bundle_scorecard_tests/files/optional-operators-cvp-common-scorecard-commands.sh

```diff
@@ -3,32 +3,37 @@
 set -o nounset
 set -o errexit
 set -o pipefail
+set -x
 
 # This value serves as a default when the parameters are not set, which should
 # only happen in rehearsals. Production jobs should always set the OO_* variable.
 REHEARSAL_BUNDLE="brew.registry.redhat.io/rh-osbs-stage/e2e-e2e-test-operator-bundle-container:8.0-3"
 OO_BUNDLE="${OO_BUNDLE:-$REHEARSAL_BUNDLE}"
+OPENSHIFT_AUTH="${OPENSHIFT_AUTH:-/var/run/brew-pullsecret/.dockerconfigjson}"
+SCORECARD_CONFIG="${SCORECARD_CONFIG:-/tmp/config/scorecard-basic-config.yml}"
 
 # Steps for running the basic operator-sdk scorecard test
 # Expects the standard Prow environment variables to be set and
 # the brew proxy registry credentials to be mounted
 
 NAMESPACE=$(grep "install_namespace:" "${SHARED_DIR}"/oo_deployment_details.yaml | cut -d ':' -f2 | xargs)
 
-OPERATOR_DIR="${ARTIFACT_DIR}/test-operator-basic"
+pushd "${ARTIFACT_DIR}"
+OPERATOR_DIR="test-operator-basic"
 
 echo "Starting the basic operator-sdk scorecard test for ${OO_BUNDLE}"
 
 echo "Extracting the operator bundle image into the operator directory"
 mkdir -p "${OPERATOR_DIR}"
 pushd "${OPERATOR_DIR}"
-oc image extract "${OO_BUNDLE}" --confirm -a /var/run/brew-pullsecret/.dockerconfigjson
+oc image extract "${OO_BUNDLE}" --confirm -a "${OPENSHIFT_AUTH}"
+chmod -R go+r ./
```

Comment:
please explan why do we need to call chmod here

Reply:
When `oc image extract ` extracts the operator bundle, it sets the permissions of the files to `0600`.  We run the scorecard test in a container, and the UUID for that container is not the owner.  Therefore, we need to increase the permissions so the container can access the bundle data.

Reply:
> We run the scorecard test in a container

I assume you meant (DCI) will run the script in k8s/openshift container as unprivileged user ...
Because CVP does not run the script.

And such explanation should be in the script as the comment